languages/rust/exercises/concept/enums/tests/enums.rs

```diff
@@ -0,0 +1,63 @@
+use enums::{output_for_short_log, parse_log_level, LogLevel};
+
+#[test]
+fn parse_error() {
+    assert_eq!(LogLevel::Error, parse_log_level("[ERROR]: Disk full"));
+}
+
+#[test]
+#[ignore]
+fn parse_warning() {
+    assert_eq!(
+        LogLevel::Warning,
+        parse_log_level("[WARNING]: Timezone not set")
+    );
+}
+
+#[test]
+#[ignore]
+fn parse_info() {
+    assert_eq!(LogLevel::Info, parse_log_level("[INFO]: Timezone changed"));
+}
+
+#[test]
+#[ignore]
+fn parse_unknown() {
+    assert_eq!(LogLevel::Unknown, parse_log_level("[FATAL]: Crash!"));
```

Comment:
```suggestion
    assert_eq!(LogLevel::Unknown("FATAL".into()), parse_log_level("[FATAL]: Crash!"));
```

I don't think it's appropriate to just discard the intended log level in the unknown case.